background/setBackground.js

```diff
@@ -0,0 +1,5 @@
+const setBackground = (weatherIcon) => {
+	const background = document.getElementsByClassName("background")[0];
+	background.style.backgroundImage = `url(${backgroundPath[weatherIcon.weatherCode]})`;
+	background.style.backgroundColor = '#efefef';
```

Comment:
In general you should avoid setting multiple properties on DOM elements - this for ex will cause 2 redraws instead of one, if you copied the element, set the styles on the copied one and replaced the current with copied and changed.